src/components/schedule-visualizer/tests/robot-trajectories-overlay.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { createMount } from '@material-ui/core/test-utils';
+import React from 'react';
+import { Map as LMap } from 'react-leaflet';
+import RobotTrajectoriesOverlay, {
+  RobotTrajectoriesOverlayProps,
+} from '../robot-trajectories-overlay';
+import { Conflict, Trajectory } from '../../../robot-trajectory-manager';
+import ColorManager from '../colors';
+import FakeTrajectoryManager from '../../../mock/fake-traj-manager';
+import { mapBound } from '../../../stories/baseComponents/utils';
```

Comment:
Hmm.. not sure if this `mapBound` should be in `src/util` since it's been used here, instead of being located in `src/stories` and import it from there.

Reply:
I created a new mapbound in the test itself since it is only for this particular test module